livekit-agents/livekit/agents/tts/tts.py

```diff
@@ -839,13 +836,13 @@ async def _decode_task() -> None:
                                 format=self._mime_type,
                             )
                             decode_atask = asyncio.create_task(_decode_task())
-
                         audio_decoder.push(data)
                     elif audio_decoder and decode_atask:
                         if isinstance(data, AudioEmitter._FlushSegment):
                             audio_decoder.end_input()
                             await decode_atask
                             _flush_frame()
+                            audio_decoder = None
```

Comment:
this was missed.. we need to reset audio decoder after each flush, otherwise the new content after flush won't be decoded